config/35-pilot.ini

```
@@ -0,0 +1,39 @@
+;===================================================================
+;                          Pilots
+;===================================================================
+
+; For each pilot type, add a new pilot section.
+; Each pilot name must be unique for the entire grid, so make sure to not
+; pick anything generic like "MAIN".
+
+; This data is used to determine where OSG pilot jobs can be sent, so it's
+; important to keep it up to date.
+
+
+;[Pilot PILOT_NAME]
+;; The number of cores for this pilot type.
+;cpucount = 1
+;; The amount of memory (in megabytes) for this pilot type.
+;ram_mb = 2500
+;; This is a whole node pilot; cpucount and ram_mb are ignored if this is true.
```

```suggestion
;; This is a whole node pilot; cpucount and ram_mb are ignored if this is true.
```

We don't properly support GPUs for whole node jobs yet but eventually I think this means that we'll also eventually ignore `gpucount`: https://opensciencegrid.atlassian.net/browse/HTCONDOR-103

---

OK; I can change this when we do that. By "are ignored" I mean their corresponding attributes don't make it into OSG_ResourceCatalog, but we don't want to do that for GPUs yet.

---

... I should make sure condor_ce_info_status is OK with those attributes being missing...

---

It is.